testdata/scripts/generated.txt

```diff
@@ -0,0 +1,27 @@
+cp orig.go.golden foo.go.golden
+exec gofmt -w foo.go.golden
+
+cp orig.go.golden foo.go
+gofumpt -w foo.go
+cmp foo.go foo.go.golden
+
+cp orig.go.golden foo.go
+gofumpt -w .
+cmp foo.go foo.go.golden
+
+-- orig.go.golden --
```

Comment:
might I suggest slightly better names for these:

* `foo.go` here, for the main input file
* `foo.go.orig` as the copy that never gets modified
* `foo.go.golden` as the version that both gofmt and gofumpt should end up with

Reply:
I think those are easier to follow, because it's clear they are all versions or copies of the same `foo.go` source code.

Reply:
thanks! PTAL